codespell_lib/data/dictionary.txt

```diff
@@ -32022,6 +32022,7 @@ ipmrovement->improvement
 ipmrovements->improvements
 ipmroves->improves
 ipmroving->improving
+iproving->improving
```

Comment:
Time permitting, I would suggest adding all verbal forms:
```suggestion
iprove->improve, I prove,
iproved->improved, I proved,
iproves->improves
iproving->improving
```

Reply:
Extended, thanks for suggestion